build.gradle

```diff
@@ -27,8 +27,8 @@ android {
     useLibrary 'org.apache.http.legacy'
 
     defaultConfig {
-        versionCode 710
-        versionName "1.52.1"
+        versionCode 711
+        versionName "1.54.0"
```

Comment:
Shouldn't it be 1.53.0 first? In general, odd numbers are prereleases and even numbers are releases, i.e. you make a 1.53.0, let test for a week or so, and then make 1.54.0 if things work fine. Or do we want to go directly to the stores with this?

Reply:
yes, plan is directly going for the store, if things get bad we can make a 1.54.x as usual, the store takes some days to approve and we can control rolling, no need to do pre-release only,

I am going for 1.54.0 just to match with desktop numbering

Reply:
yip, and one can rollout slowly. "real test-only releases" are done only where it was clear from the beginning on, that this will never hit the store. but yeah, there is always quite some confusion about versioning, there is not really a documentation or so about that :)